kernel/kernel-impl/src/main/webapp/WEB-INF/db-components.xml

```diff
@@ -463,11 +437,9 @@
 		</property>
 		<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>
 	</bean>
-	-->
 
 	<!--  The "Global" JPA Entity Manager Factory -->
-	<!-- uncomment this bean for EntityManager
-	<bean id="org.sakaiproject.springframework.orm.jpa.EntityManagerFactory" class="org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean">
+	<bean id="entityManagerFactory" class="org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean">
```

Comment:
Id renaming to fit with Spring Boot requirements:

Consider defining a bean named 'entityManagerFactory' in your configuration.